Makefile

```diff
@@ -1,12 +1,22 @@
 all: test
 
-uninstall:
-	pip freeze | grep -v "^-e" | xargs pip uninstall -y
-
-install:
+install.tools:
```

Comment:
The naming convention for Make targets [seems to be](https://stackoverflow.com/questions/32130664/is-there-a-naming-convention-for-makefile-targets-and-variables) using hyphens instead of dots.